website/package.json

```diff
@@ -47,5 +47,6 @@
       "last 1 safari version"
     ]
   },
-  "description": "RISC Zero - The ZKVM"
+  "description": "RISC Zero - The ZKVM",
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
is this change intended?

Reply:
no - must have been auto generated following https://github.com/risc0/risc0/tree/main/website#local-development. Is there a reason we want to let that yarn version float?
 (TIL https://github.com/yarnpkg/berry/discussions/5394  can help you set project and/or global versions)